modules/internal/ChapelRange.chpl

```diff
@@ -102,7 +102,7 @@ module ChapelRange {
   pragma "plain old data"
   pragma "range"
   @chpldoc.nodoc
-  record range
+  record _range
   {
```

Comment:
Do functions like the following also need to change?

```chapel
proc range.strType type do return chpl__rangeStrideType(idxType);
```

Reply:
I guess also along these lines, does the documentation links still work?

Reply:
> Do functions like the following also need to change?
> 
> ```chapel
> proc range.strType type do return chpl__rangeStrideType(idxType);
> ```

I guess not (I'll confess I don't fully I understand why not but we must have some magic making it work for these "internal types"). I feel confident I'm not breaking anything though because:

- We have functions like `inline proc locale.hostname` where we similiarly have `locale` as a reserved word and have a `_locale` function.
- If this was a breaking change I'd expect to see a bunch of paratests failures about it but I don't.

> I guess also along these lines, does the documentation links still work?

Good question, I didn't think to double check that I didn't break the doc.

I've built it on my branch and taken a look and it looks the same as what we have live.